### ISSUE TITLE
Unity projects targeting netstandard 2.0 have busted intellisense

### ISSUE THREAD

**Issue by Therzok** [mono/monodevelop#5747]
• Create new Unity project in 2018.2
• Set the player settings to .net 4.x, api compat level to .net standard 2.0 (file -> Build settings -> Player Settings -> that opens a window inside Unity, check in Other Settings)
• Add a C# script
• Open in VSfM
• Notice you don’t have intellisense and everything has squigly lines (edited)

csproj excerpt:
```
  <PropertyGroup>
    <NoConfig>true</NoConfig>
    <NoStdLib>true</NoStdLib>
    <AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
    <ImplicitlyExpandNETStandardFacades>false</ImplicitlyExpandNETStandardFacades>
    <ImplicitlyExpandDesignTimeFacades>false</ImplicitlyExpandDesignTimeFacades>
  </PropertyGroup>
```

From JB: it says that the typeforward for System.Object in .netstandard2.0 creates a cycle

Possibly we need to support ImplicitlyExpandNETStandardFacades

> VS bug [#669795](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/669795)

**Comment by mhutch** [comment]
Yet another issue that would be fixed by ResolveReferences (or a real design-time build) 😐

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-24] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #5804 merged by: Lluis Sanchez  
 Author: mono  
 Commit: b49f3df39472b4f7488a7597f9d359d2e639a8a2 (xamarin/monodevelop)
